src/android/OSLCNOError.java

```diff
@@ -0,0 +1,37 @@
+package de.appplant.cordova.plugin.localnotification;
+
+import org.json.JSONException;
+import org.json.JSONObject;
+
+public class OSLCNOError {
+
+    private final String code;
+    private final String message;
+
+    public String getCode() { return code; }
+    public String getMessage() { return message; }
+
+    public OSLCNOError(String code, String message) {
+        this.code = code;
+        this.message = message;
+    }
+
+    static OSLCNOError EXACT_PERMISSION_ERROR =
+            new OSLCNOError("OSLCNO-001","Unable to schedule an exact alarm due to lack of permissions.");
+
+    static OSLCNOError EXACT_PERMISSION_WARNING =
```

Comment:
should we create a separate "enum" for warnings? so it's easier to pass that information to the wrapper

Reply:
Sounds good!. Will update.